### PR TITLE
Move logger

### PR DESCRIPTION
**Pull request by NishanthSpShetty**
# Description

Update logger to `netbookai/log` 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# why

Use of zap logger provides limited set of capability, so using `netbookai/log` 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/spawnersvc/main.go [hunk 1]
```diff
@@ -37,23 +38,22 @@ func startHttpServer(g *group.Group, config config.Config, logger *zap.SugaredLo
 	router.Handle("/metrics", promhttp.Handler())
 
 	g.Add(func() error {
-
-		logger.Infow("startHttpServer", "transport", "debug/HTTP", "address", address)
+		logger.Error(ctx, "startHttpServer", "transport", "debug/HTTP", "address", address)
```
Comment (sid255) [inline cmd/spawnersvc/main.go:41]: Why logging at Error level?

**Review by sid255 — CHANGES_REQUESTED** [review]
LGTM

File: cmd/spawnersvc/main.go (hunk 1 above)
Comment (NishanthSpShetty) [reply cmd/spawnersvc/main.go]: updated to Info.

**Review by sid255 — APPROVED** [review]
LGTM

**Pull request merged**